argus/util.py

```diff
@@ -65,6 +66,7 @@
     OPEN_NEBULA_SERVICE: "",
     MAAS_SERVICE: "",
     CONFIG_DRIVE_SERVICE: "",
+    NO_SERVICE: "",
```

Comment:
no need for this line, as there is no need for metadata service url
also, make sure you don t have in the config files a separate section for the metadata url, if the metadata service type does not have an url defined